mentat/code_feature.py

```diff
@@ -180,6 +180,28 @@ def count_tokens(self, model: str) -> int:
         code_message = self.get_code_message()
         return count_tokens("\n".join(code_message), model, full_message=False)
 
+    @staticmethod
+    def from_string(string: str) -> CodeFeature:
+        """
+        Create a CodeFeature from a string.
+        """
+
+        print("in CodeFeature.from_string", string)
+
+        # find last colon
+        colon_index = string.rfind(":")
```

Comment:
The windows tests are failing because their paths start with `C:`. In general its not safe to split on ":" because paths can contain that character.

You can use `get_code_feratures_for_path` from `mentat/include_files.py` to do this.

Reply:
Ah perfect, I was imagining you already have something for this but didn't find it when I first wrote the code